src/main/java/core/basesyntax/service/TransactionStrategy.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.service;
+
+import core.basesyntax.FruitTransaction;
+import core.basesyntax.db.DataBase;
+
+public class TransactionStrategy {
+    private static int balanceFruits = 0;
+    private static int purchaseFruits = 0;
+    private static int returnFruits = 0;
+    private static int supplyFruits = 0;
+
+    public void getShopService() {
```

Comment:
that's now how we do strategy
it should be a map, operation -key, dedicated handler - value 
we don't do calculations here directly
Long chains of if`s, switches should be avoided if we can use simple iteration, or define a mapping

Reply:
dont fixed
try to google at first, how Strategy pattern implements
if it will not help - come to Mate chat or QnA